docs/workstation_release_management.rst

```diff
@@ -131,8 +131,100 @@ Step 7: Deploy the package to ``apt-prod``
 Release an RPM package
 ======================
 
-Release ``securedrop-workstation-dom0-config``
-----------------------------------------------
+Release Process for ``securedrop-workstation-dom0-config``
+----------------------------------------------------------
+
+Step 1: Create a release candidate (rc) tag
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Create a release branch in the ``securedrop-workstation`` repository.
+2. Push a changelog commit.
+3. Push an rc tag in the format ``<major>.<minor>.<patch>~rcN`` on your new commit. 
+   We will be building from this tag in the next step.
+
+Step 2: Build and deploy the package to ``yum-test``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Run ``update_version.sh <major>.<minor>.<patch>~rcN``.
+2. Manually update the changelog in the spec file (``rpm-build/SPECS``).
+3. Manually cut a <major>.<minor>.<patch>~rcN tag (must be signed, even
+   if it's with a personal key).
+4. Create a pull request into the ``release/<major>.<minor>.<patch>~rcN`` branch for 
+   N>1 (otherwise it's just the first push of release/<major>.<minor>.<patch>).
+5. Build from this tag in a clean environment, then create a pull request for the
+   ``securedrop-yum-test`` repository and commit the logs to the ``securedrop-build-logs`` 
+   repository.
+6. Once the pull request is merged, QA can begin.
+
+Step 3: Begin QA
+~~~~~~~~~~~~~~~~
+
+You can start the QA process on the rc package that you deployed to
+https://yum-test.freedom.press. If a bug is found, the fix can be added to the release
+branch and you can begin the rc process all over again with a new rc version.
+
+Once an rc package has been approved, you are ready to move on to the next step.
+
+Step 4: Create a release tag
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Begin this step on the day you want to release the package. It's best to start this
+process early in the day to ensure there is enough time for final QA checks,
+signing ceremonies, and stakeholder communications.
+
+1. Push a release tag on the same commit of the rc tag that was approved during QA.
+2. :ref:`Sign the tag with the SecureDrop release key` (or ask another maintainer to do this).
+
+Step 5: Build and deploy the package to ``yum-qa``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once the package on ``yum-test`` has passed QA and it appears that no additional
+release candidates are needed, repeat the process from Step 2, this time opening
+the pull request against ``securedrop-yum-qa``. Once the package is uploaded there,
```

Comment:
There is no securedrop-yum-qa, I think. The preflight procedure as I understand it is the same as for the repos:
- create and sign prod RPM
- create a `release` branch on the `securedrop-yum-prod` repo and add the RPM there
- submit a PR for that branch - a webhook will deploy it to yum-qa.securedrop.org
- do preflight checks (as described below)
- when good, merge to `main`, which will deploy it to prod.